tests/D2L.CodeStyle.Analyzers.Test/Immutability/MutabilityInspectorTests.cs

```diff
@@ -551,6 +552,67 @@ sealed class Baz { }
 			AssertUnauditedReasonsResult( ty, "ItsUgly", "ItHasntBeenLookedAt" );
 		}
 
+		[Test]

```

Comment:
Could you add these to the spec tests?

Reply:
Along side, or instead of?

Reply:
Either works for me. I'm surprised we didn't have a test case covering this already! (but asserting there was a diagnostic.)